tests/test_cliCmdQueryDatasetTypes.py

```diff
@@ -149,6 +149,99 @@ def testQueryDatasetTypes(self):
             self.assertEqual(result.exit_code, 0, clickResultMsg(result))
             self.assertIn("No results", result.output)
 
+    def testRemoveDatasetTypes(self):
+        self.maxDiff = None
+        datasetName = "test"
+        instrumentDimension = "instrument"
+        visitDimension = "visit"
+        storageClassName = "StructuredDataDict"
+        runner = LogCliRunner()
+        with runner.isolated_filesystem():
+            result = runner.invoke(cli, ["create", "here"])
+            self.assertEqual(result.exit_code, 0, clickResultMsg(result))
+            for name in (
+                datasetName,
+                "testA",
+                "testB",
+                "testC",
+                "testD",
+                "other",
+                "another",
+                "option",
+                "option2",
+                "placeholder",
+            ):
+                # Create the dataset type.
+                result = runner.invoke(
+                    cli,
+                    [
+                        "register-dataset-type",
+                        "here",
+                        name,
+                        storageClassName,
+                        instrumentDimension,
+                        visitDimension,
+                    ],
+                )
+
+            # checks wildcard / literal combination
```

Comment:
Python prefers full sentences for code comments. This would be something like:

```python
# Check wildcard / literal combination.
```

Same comment for the comments below.